src/firecrest/status/health_check/checks/health_check_s3.py

```diff
@@ -16,8 +16,8 @@
 
 class S3HealthCheck(HealthCheckBase):
 
-    def __init__(self, system: HPCCluster, timeout: int):
-        super().__init__(system)
+    def __init__(self, timeout: int):
+        super().__init__(None)
```

Comment:
Just a comment, we can remove the `None` here

Reply:
Yes, thanks